src/js/addons/widgets/DropDownMenu.js

```diff
@@ -2,49 +2,52 @@ import Widget from "./Widget.js"
 
 import { getAddressSceneContent } from '../../core/codemirror/yaml-tangram.js';
 
-export default class DropDownMenu extends Widget {
-    constructor (datum) {
-        super(datum);
+let cm;
```

Comment:
Let's avoid having global variables on modules. Although they are isolated on the module is not clear if this will unleash weird behavior on posible multiple instances of tangram-play